sql/item_mods.sql

```diff
@@ -2303,12 +2303,12 @@ INSERT INTO `item_mods` VALUES (10788,30,5); -- MAQUETTE RING, Magic Accuracy+5
 -- INSERT INTO `item_mods` VALUES (10788,LEGION:ADD REFRESH EFFECT,1);
 INSERT INTO `item_mods` VALUES (10789,25,-4); -- FOLHARDI RING, Accuracy-4 Attack+9
 INSERT INTO `item_mods` VALUES (10789,23,9); -- ATT
-INSERT INTO `item_mods` VALUES (10790,1,4); -- EPHEDRA RING, DEF:4 Healing magic skill +7 Enhances "Cursna" effect
-INSERT INTO `item_mods` VALUES (10790,112,7); -- HEALING SKILL
-INSERT INTO `item_mods` VALUES (10790,310,1); -- EN CURSNA
-INSERT INTO `item_mods` VALUES (10791,1,5); -- HAOMAS RING, DEF:5 Healing magic skill +8 Enhances "Cursna" effect
-INSERT INTO `item_mods` VALUES (10791,112,8); -- HEALING SKILL
-INSERT INTO `item_mods` VALUES (10791,310,1); -- EN CURSNA
+INSERT INTO `item_mods` VALUES (10790,1,4);       -- Ephedra Ring: DEF 4
+INSERT INTO `item_mods` VALUES (10790,112,7);     -- Healing magic skill +7
+INSERT INTO `item_mods` VALUES (10790,310,10);    -- Cursna effect +10%
+INSERT INTO `item_mods` VALUES (10791,1,5);       -- Haoma's Ring: DEF 5
```

Comment:
Question: do you have something automating comment alignment? There's no standard in the project for the commenting (some are even of the mind it doesn't matter and willing to just throw it away, where I see it valuable for not having to run to modifier.h every other minute).

I have no preference on it being aligned or not but folks are going to have to come together on wther to try and multi space align or to single space the commenting, so that people aren't changing it back and forth every other PR.

@Ninjistix @wrenffxi @R3P0FFXI  
-tag!-